hugo/content/docs/recipes/lexis/case-insensitive-languages.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Case-insensitive languages
+weight: 100
+---
+
+Case-insensitive languages are more suitable for beginners, because they don't have to worry about the case of identifiers and keywords.
+
+In Langium, there are basically two options you can choose from:
+
+* you can either make Langium case-insensitive by configuration
+* or you can include case-insensitivity only where you need it
+
+In both ways you need to take care of identifiers and cross-references.
```

Comment:
Suggestion: Given that the second one only applies to terminals and not keywords, I wouldn't say they're equal alternatives:
```suggestion
In case you want to provide your users the same flexibility with your language, there are different levels of case-insensitivity in Langium:

* You can make Langium's parser completely case insensitive using the language configuration
* You can include case-insensitivity for specific terminal rules
* You can make cross references case insensitive

All of these options can be enabled independent of one another.
```

Reply:
I like especially the last sentence :D Thank you